### DISCUSSION TITLE
Broken in Vivaldi (Chromium)

### DISCUSSION THREAD

**Discussion by max06** [HafisCZ/sf-tools#11] [General]
https://github.com/HafisCZ/sf-tools/blob/fbcc87eb382f6604de8072c6763e3252cae83d82/inventory.html#L233

Beta shows me this when I try to use the inventory. Latest Vivaldi on Windows. Other pages like the dungeon sim are working perfectly fine.

Not sure why this is not working any more, but it would be great to have it back.

**Comment by HafisCZ** [comment]
Hi, can you tell me what is going on in the console when this happens? I can't see any errors in Sentry (most likely due to Vivaldi 😅)

**Reply by max06** [reply]
That might be due to tracking blockers, sorry 😅 

Actually, it seems to be solved. It just showed me "Updating database to new version" after opening the inventory in the console. No more error. It could have been a caching issue...

Happy to provide more insights if you want to dig into that rabbit hole 🐰

**Reply by HafisCZ** [reply]
Ah great 👌 yeah I was touching things so maybe one-off or got resolved in the meantime
